src/signals/incident-management/containers/Dashboard/charts/area-chart.test.ts

```diff
@@ -1,17 +1,18 @@
 // SPDX-License-Identifier: MPL-2.0
```

Comment:
I see these test files  are kebab cased. But we only use that fore utils or definitions. Not for test files right?

Reply:
yeah I wasn't sure. The file is named area-chart. Because it was a constant at first, but then I changed it to be a function because it needed arguments. I will rename it to `getAreaChartSpec`, since that is what the function is called.